sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -857,6 +857,31 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     assert(sql(s"list file $testFile").count() == 1)
   }
 
+  test("ADD ARCHIVE command") {
+    withTempDir { dir =>
+      val file = File.createTempFile("someprefix1", "somesuffix1", dir)
+      val zipFile = new File(dir, "test.zip")
+      TestUtils.createJar(Seq(file), zipFile)
+      sql(s"ADD ARCHIVE ${zipFile.getAbsolutePath}#foo")
```

Comment:
Could you add a test for `ADD ARCHIVES`, too?

Reply:
What if the given file does not exist? Also, what if the given file has a unsupported format?

Reply:
> Could you add a test for ADD ARCHIVES, too?

It's not supported.

> What if the given file does not exist? Also, what if the given file has a unsupported format?

Covered such cases.